src/pages/profilePage/ProfilePage.tsx

```diff
@@ -0,0 +1,14 @@
+import { Link } from 'react-router-dom'
+
+import './profilePage.css'
+
+export function ProfilePage() {
+  const LogOut = () => sessionStorage.clear()
+  return(
+    <div className='profile-page'>
+      <Link to='/home'>
```

Comment:
mozes procitati vise o linkovima [Ovde](https://javascript.plainenglish.io/react-router-dom-the-differences-between-navlink-link-and-a-bb5c27a48bfc)